tests/transpilers/test_python.py

```diff
@@ -599,3 +639,117 @@ def test_transpiler_while_expr() -> None:
     assert (
         generated_code == expected_code
     ), f"Expected '{expected_code}', but got '{generated_code}'"
+
+
+def test_transpiler_ifexpr_without_else() -> None:
+    """Test astx.IfExpr without else block."""
+    # determine condition
+    cond = astx.BinaryOp(
+        op_code=">", lhs=astx.LiteralInt32(1), rhs=astx.LiteralInt32(2)
+    )
+
+    # create then block
+    then_block = astx.Block()
+
+    # define literals
+    lit_2 = astx.LiteralInt32(2)
+    lit_3 = astx.LiteralInt32(3)
+
+    # define operation
+    op1 = lit_2 + lit_3
+
+    # Add statement to the then block
+    then_block.append(op1)
+
+    # define if Expr
+    if_expr = astx.IfExpr(condition=cond, then=then_block)
+
+    # Initialize the generator
+    generator = astx2py.ASTxPythonTranspiler()
+
+    # Generate Python code
+    generated_code = generator.visit(if_expr)
+
+    # Expected code for the binary operation
+    expected_code = "    (2 + 3) if  (1 > 2)"
```

Comment:
in the case when there is no "else", it should return "None"

something like this:

```suggestion
    expected_code = "    (2 + 3) if  (1 > 2) else None"
```